packages/@aws-cdk/aws-s3-deployment/test/lambda/aws

```diff
@@ -15,8 +15,12 @@ import shutil
 
 scriptdir=os.path.dirname(os.path.realpath(__file__))
 
-# if "cp" is called with a local destination, copy a test zip file to the destination or 
-if sys.argv[2] == "cp" and not sys.argv[4].startswith("s3://"):
+# if "cp" is called to contents, copy a test zip file to contents
+# else if "cp" is called with a local destination, copy a test zip file to the destination or
```

Comment:
This is kind of a nit, but these comments are the wrong order for what's the `if` statement and what's the `else if` statement.

Reply:
I attempted to correct this. Let me know if the order makes more sense now.